surfactant/cmd/tui.py

```diff
@@ -452,17 +554,19 @@ def __init__(self):
         super().__init__()
         self.generate_tab = GenerateTab()
         self.merge_tab = MergeTab()
+        self.plugin_settings_tab = PluginSettingsTab()
         self.context_tab = ContextTab()
 
     def compose(self) -> textual.app.ComposeResult:
         yield textual.widgets.Header()
         yield textual.widgets.Footer()
-        Tab = textual.widgets.Tab
         with textual.widgets.TabbedContent():
             with textual.widgets.TabPane("Generate"):
                 yield self.generate_tab
             with textual.widgets.TabPane("Merge"):
                 yield self.merge_tab
+            with textual.widgets.TabPane("Plugin Settings"):
+                yield self.plugin_settings_tab
             with textual.widgets.TabPane("Context"):
                 yield self.context_tab
```

Comment:
```suggestion
            with textual.widgets.TabPane("Context"):
                yield self.context_tab
            with textual.widgets.TabPane("Settings"):
                yield self.plugin_settings_tab
```

Renaming the tab to just Settings, since it also includes "core" settings. Slight reordering of the tabs to place it at the end.